Cloud Application Manager/Data Center Edition/camdce-with-aws-master-account.md

```diff
@@ -9,50 +9,50 @@
 
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Additional information](#additional-information)
-* [Create the Policy for the EC2 Instance Role](#create-the-policy-for-the-ec2-instance-role)
-* [Create the EC2 Instance Role](#create-the-ec2-instance-role)
-* [Attach the IAM Role to the Cloud Application Manager Data Center Edition EC2 Instance](#attach-the-iam-role-to-the-cloud-application-manager-data-center-edition-ec2-instance)
-* [Enable the EC2 Instace Role on CAM](#enable-the-ec2-instace-role-on-cam)
-* [Create a Policy and Role for the managed account](#create-a-policy-and-role-for-the-managed-account)
-* [Create a Cloud Application Manager AWS Provider](#create-a-cloud-application-manager-aws-provider)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#Overview)
```

Comment:
Hey Rob, latest MD plugin creates the ToC links with Capital Case, but we need them to be lowercase for them to work with the CTL MD engine. Please, leave them in lowercase (ask Denes about how to downgrade the plugin, if you want). Also, use "*" as bullet points